ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-stable-4.13-loaded-upgrade-from-stable-4.12.yaml

```diff
@@ -0,0 +1,86 @@
+build_root:
+  image_stream_tag:
+    name: ci-tools-build-root
+    namespace: ci
+    tag: latest
+images:
+- dockerfile_path: prow/Dockerfile
+  to: ocp-qe-perfscale-ci-loaded-upgrade-412to413
+promotion:
+  namespace: ci
+  tag: latest
+releases:
+  intermediate:
+    release:
+      channel: candidate
+      version: "4.12"
+  latest:
+    release:
+      channel: candidate
+      version: "4.13"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: perfscale-rosa-multiaz-6nodes-stage-loaded-upgrade412
+  cron: 0 8 * * 3
+  steps:
+    cluster_profile: aws-perfscale-qe
+    env:
+      CHANNEL_GROUP: candidate
+      GC: "false"
+      IF_MINOR_Z_UPGRADE: "true"
+      MULTI_AZ: "true"
+      OPENSHIFT_VERSION: 4.12.26
+      REGION: us-east-2
```

Comment:
@liqcui Is there any reason to test in `us-east-2`? Can we move to `us-west-2` just like all our other tests?

Reply:
Change to us-west-2